ivy-rich.el

```diff
@@ -378,7 +394,25 @@ or /a/…/f.el."
                (ffip-project-root)))
             ((require 'project nil t)
              (when-let ((project (project-current nil dir)))
-               (car (project-roots project))))))))
+               (car (project-roots project)))))))
+
+(defun ivy-rich-switch-buffer-root (candidate)
+  (when-let* ((dir (ivy-rich--switch-buffer-directory candidate)))
+    (let ((cached-value (if ivy-rich-project-root-cache-mode
+                            (gethash dir ivy-rich-project-root-cache 'not-found)
+                          'not-found)))
+      (if (not (eq cached-value 'not-found)) cached-value
```

Comment:
Putting `cached-value` in an individual newline seems clearer here.

Reply:
Done